### DISCUSSION TITLE
How to fine-tune Arabert sentiment-analysis with neutral output?

### DISCUSSION THREAD

**Discussion by wirokemproh** [aub-mind/arabert#76] [Q&A]
I'm trying to finetune Arabert for Sentiment Analysis. So I'm following this [example](https://colab.research.google.com/drive/1P9iQHtUH5KUbTVtp8B4-AopZzEEPE0lw?usp=sharing)

However, it seems that the output of the model is only POSITIVE and NEGATIVE.
I also want to have NEUTRAL on the output.

How can I add this output to the model?

**Comment by WissamAntoun** [comment]
Weird, are you sure the training and testing set contains Neutral examples? Because this notebook supports more than 2 labels, and it works with datasets like ArsenTD ...
